tools/moFF/moff.xml

```diff
@@ -192,24 +192,24 @@
                --rt_peak_win $task.rt_peak_win
                --rt_peak_match $task.rt_peak_match
                --output_folder ./out
-               #if $task.peptide_summary:
+               #if ($task.peptide_summary):
                    --peptide_summary
                #end if
-               #if $task.match_filter.match_filter_selector:
+               #if ($task.match_filter):
                     --match_filter 
                     --quantile_thr_filtering $task.sample_size
 					--sample_size $task.quantile_thr_filtering
-					#if $task.custom_ptm_file.custom_ptm_file_selector:
-					--ptm_file $task.ptm_file   
+					#if ($task.ptm_file):
```

Comment:
@bgruening  here I should have a syntax error I because i got this error :  'Some #directives are missing their corresponding #end ___ tag: if
Line 154, column 4

What am I doing wrong ? 
Another question is with  $task.variable_name I get the value of parameter right ? 
Line|Cheetah Code
----|-------------------------------------------------------------
151 |               mv ./out/peptide_summary_intensity_moFF_run.tab '$output_peptide_summary'
152 |              #end if
153 |      #end if
154 |
        ^
'